jobmaster/system/interface.go

```diff
@@ -4,18 +4,17 @@ import (
 	"context"
 
 	"github.com/hanfei1991/microcosm/model"
+	"github.com/hanfei1991/microcosm/pkg/metadata"
 )
 
 // JobMaster maintains and manages the submitted job.
 type JobMaster interface {
 	// DispatchJob dispatches new tasks.
-	DispatchTasks(ctx context.Context, tasks []*model.Task) error
+	DispatchTasks(tasks ...*model.Task)
 	// Start the job master.
-	Start(ctx context.Context) error
+	Start(ctx context.Context, metaKV metadata.MetaKV) error
```

Comment:
the meta server / message server should be inited when creating the jobmaster /jobmanager/task.

Reply:
theoretically it should be. I will leave a comment